static/tipuesearch/tipuesearch.css

```diff
@@ -45,8 +45,9 @@ http://www.tipue.com/search
 }
 #tipue_search_replace, #tipue_search_error, .tipue_search_content_text
 {
-     font: 300 16px/1.6 'Open Sans', sans-serif;
-	color: #666; 
+     font: 300 1.25rem "Ubuntu", "PT Serif", Georgia, sans-serif;
```

Comment:
search result can be a little bit different from home page with posts and smaller font more suitable in this case, let's revert only font size to original for tipuesearch `16px/1.6` - what do you think?